tests/functional/tests/smoke.py

```diff
@@ -146,6 +146,7 @@ def generate_expected_plan(self, backup: bool = True) -> str:
             "\t\t\tUpgrade 'designate-bind' from 'ussuri/stable' to the new channel: "
             "'victoria/stable'\n"
             "\t\t\tWait for up to 300s for app 'designate-bind' to reach the idle state\n"
+            "\t\t\tWait for up to 300s for app 'designate-bind' to reach the idle state\n"
```

Comment:
Why are we having a duplicate here?

Reply:
Because of how everything is split into various methods - I think the post upgrade steps always puts in a wait.  From the upgrade steps I can't really communicate nicely with the post upgrade steps to deduplicate.

Reply:
I think now I see the reason. Channel based apps like `designate-bind` upgrades just by changing the channel so you'll have this duplication because it's a special case where there is not `_get_change_install_repository_step` and `_get_units_upgrade_steps`.